internal/web/server.go

```diff
@@ -468,6 +498,49 @@ func defaultKubeconfigExists() bool {
 	}
 }
 
+func (s *server) initInformer() (cache.Store, cache.Controller) {
+	return cache.NewInformer(
+		&cache.ListWatch{
+			ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
+				var pkgList v1alpha1.PackageList
+				err := s.pkgClient.Packages().GetAll(context.Background(), &pkgList)
+				return &pkgList, err
+			},
+			WatchFunc: func(options metav1.ListOptions) (watch.Interface, error) {
+				return s.pkgClient.Packages().Watch(context.Background())
+			},
+		},
+		&v1alpha1.Package{},
+		0,
+		cache.ResourceEventHandlerFuncs{
+			AddFunc: func(obj interface{}) {
```

Comment:
I think it is a little more modern to use `any` instead of `interface{}`. It's just an alias.